### DISCUSSION TITLE
Cfclient scan does not find a Crazyflie

### DISCUSSION THREAD

**Discussion by ViniCast** [bitcraze/discussions#1081] [Q&A] [answered]
Hello!
I have installed cfclient on Win11 and I have a Crazyflie PA radio to connect with the drone. When I scan, the cfclient does not shows any interface to connect. I have never modified the crazyflie default address. This is whaat the cmd screen shows:

UserWarning: Using SDL2 binaries from pysdl2-dll 2.24.0
INFO:cfclient.gui:Disabling STL printouts
INFO:cfclient.utils.input.inputreaders:Input readers: ['linuxjsdev', 'pysdl2']
INFO:cfclient.utils.input.inputreaders:Could not initialize [linuxjsdev]: Only supported on Linux
INFO:cfclient.utils.input.inputreaders:Successfully initialized [pysdl2]
INFO:cfclient.utils.input.inputinterfaces:Found interfaces: ['leapmotion', 'wiimote', 'zmqpull']
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [leapmotion]: Leap Motion library probably not installed (No module named 'leapsdk')
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [wiimote]: Missing cwiid (wiimote) driver No module named 'cwiid'
INFO:cfclient.utils.config:Dist config read from C:\Users\vinip\AppData\Local\Programs\Python\Python311\Lib\site-packages\cfclient/configs/config.json
INFO:cfclient.utils.config:Config file read from [C:\Users\vinip\AppData\Local\Bitcraze\cfclient/config.json]
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [zmqpull]: ZMQ input disabled in config file
INFO:cfclient.utils.zmq_led_driver:ZMQ led disabled in config file
INFO:cfclient.utils.zmq_param:ZMQ param disabled in config file
INFO:cfclient.utils.input:Using device blacklist [(VirtualBox|VMware)]
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cflib.crtp.radiodriver:v0.53 dongle with serial N/A found
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.utils.input.inputreaders.pysdl2:Looking for devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Found 0 devices
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for open_toolboxes
INFO:cfclient.ui.tab_toolbox:No config found for open_toolboxes
INFO:cfclient.utils.input.inputreaders.pysdl2:Looking for devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Found 0 devices
INFO:cflib.drivers.cfusb:Looking for devices....
INFO:cflib.crtp.prrtdriver:Initialized PRRT driver.
INFO:cfclient.utils.input.inputreaders.pysdl2:Looking for devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Found 0 devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Looking for devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Found 0 devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Looking for devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Found 0 devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Looking for devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Found 0 devices

**Comment by knmcguire** [comment, marked as answer]
Hi!

Seems that the CFclient has found your Crazyradio so that is not the issue... the pysdl2 warning seems to be a constant for windows even with a fully functional cfclient so you can ignore that

You could try to connect with the Crazyflie with a USB cable? Since you are using Windows you would need to use the same[ Zadig instructions ](https://www.bitcraze.io/documentation/repository/crazyradio-firmware/master/building/usbwindows/) for the Crazyflie as well. Then you can see in the connect>configure 2.X if the address is indeed different or not.

**Reply by amitx64** [reply]
@ViniCast, I wanted to update you on my progress. I've successfully set up the `python virtual environment` for `cfclient` on Windows, and it's working well. In line with @knmcguire's advice, I'm following the same `Zadig instructions` for the `Crazyflie`, given that I'm using Windows. Just keeping you in the loop on the latest developments.

**Reply by ViniCast** [reply]
I did exactly what @knmcguire said and now it's working. For some reason the address was different from default

**Reply by knmcguire** [reply]
good it is working! next time you can select my answer so that others know that it has been solved ;)
